test/charts/progress/progress.spec.js

```diff
@@ -0,0 +1,49 @@
+describe('Directive: pfProgressChart', function() {
+  var $scope, $compile, element, progressBar, title, subTitle;
+
+  beforeEach(module(
+    'patternfly.charts',
+    'charts/progress/progress-chart.html'
+  ));
+
+  beforeEach(inject(function(_$compile_, _$rootScope_, _$timeout_) {
+    $compile = _$compile_;
+    $scope = _$rootScope_;
+    $timeout = _$timeout_;
+  }));
+
+  beforeEach(function() {
+
+    $scope.data = {
+      'used': '8',
+      'total': '16'
+    };
+
+    $scope.config = {
+      'title': 'CPU Usage',
+      'units': 'MB'
+    }
+
+  });
+
+  var compileChart = function (markup, scope) {
+    var el = $compile(markup)(scope);
+    scope.$digest();
+    return el;
+  };
+
+  it("should set the width of the inner bar to 50%", function() {
+
+    element = compileChart('<div pf-progress-chart chart-data=data config=config></div>', $scope);
+
+    progressBar = angular.element(element).find('.progress-bar').css('width');
+    expect(progressBar).toBe("50%");
+
+    title = angular.element(element).find('.progress-description').html();
+    expect(title).toBe("CPU Usage");
+
+    subTitle = angular.element(element).find('.progress-bar span').html();
+    expect(subTitle).toBe("8 of 16 MB");
+  });
+
+});
```

Comment:
Could we add a test to verify updating the data, title or units updates the displayed components? Silly and easy, but it adds check.